zombies.js

```diff
@@ -79,6 +92,24 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+function Player(name, health, strength, speed) {
+  this._pack = [];
+  this._maxHealth = health;
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
+  this.speed = speed;
+  this.isAlive = true;
+  this.equipped = false;
+}
+
+Player.prototype.getPack = function() {
+  return this._pack;
+};
+
+Player.prototype.getMaxHealth = function() {
+  return this._maxHealth;
+};
```

Comment:
The getPack and the getMaxHealth should be within the constructor function for this class and not on the prototype. This allows it to reference the private _pack and _getMaxHealth consts.